integration/utils/transactions.go

```diff
@@ -126,6 +130,73 @@ func MakeCollectionRegisterNodeTx(
 	return tx, nil
 }
 
+func MakeStakingCollectionCloseStakeTx(
```

Comment:
noting this is unused currently - we can keep it if you think we'll use in future tests

Reply:
Yes exactly in the future we want to create staking collections for all the nodes bootstrapped in the integration test network. This will come in handy.